active.go

```diff
@@ -35,9 +38,9 @@ func init() {
 
 func runActive(cmd *Command, args []string) {
 	if account == "" {
-		account, _ := Config.Load("current", "account")
-		data, _ := Config.Load("accounts", account)
-		var creds ForceCredentials
+		account, _ := salesforce.Config.Load("current", "account")
```

Comment:
Why is `Config` in `salesforce` rather than `util`? It doesn't actually communicate with the API, but rather loads values from a `Config` file that will then be used to initialize clients.